packages/eth-json-rpc-provider/package.json

```diff
@@ -45,7 +45,6 @@
     "@metamask/auto-changelog": "^3.4.3",
     "@types/jest": "^27.4.1",
     "deepmerge": "^4.2.2",
-    "depcheck": "^1.4.3",
```

Comment:
This package and `json-rpc-engine` actually have extraneous scripts and dependencies — I will make another PR to correct this.